modules/front-end/src/app/core/components/header/header.component.html

```diff
@@ -7,6 +7,7 @@
 
   <div class="header-btns">
     <div class="header-btn feedback" (click)="openFeedbackModal()" i18n="@@common.feedback">Feedback</div>
+    <a class="header-btn doc-btn" href="https://docs.featbit.co"><span nz-icon nzType="read" nzTheme="outline"></span></a>
```

Comment:
Please add target="_blank" to the link so the doc page would be open in a new tab